api/provider/aws/service_delete.go

```diff
@@ -1,5 +1,65 @@
 package aws
 
-func (s *ServiceProvider) Delete(ServiceID string) error {
+import (
+	"fmt"
+
+	"github.com/aws/aws-sdk-go/service/ecs"
+)
+
+// Delete a specified service within a cluster.
+func (s *ServiceProvider) Delete(serviceID string) error {
```

Comment:
It looks like we're missing this behavior, is it no longer needed?
https://github.com/quintilesims/layer0/blob/develop/api/backend/ecs/service_manager.go#L130